entrypoint.sh

```diff
@@ -10,13 +10,16 @@ export SERVICE_ACCOUNT_TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount
 # TODO: Read the vault path from service in future
 # a vault path should end with login. If the path is /auth/temp/login,
 # the CNAME must be auth.temp (do not end the login in end)
-SERVICE_URL="vault-path.infra.svc.cluster.local"
+if [ ! "$ENV" == "local" ]; then
```

Comment:
Replace with better variable. My choice would be `K8S_SETUP` set to `FALSE` explicitly. This is debatable though. The logic is fine.

Reply:
cool, once let the other reviewer also pitch the name. I'll change it to what everyone agree upon